sample/app/src/main/java/com/example/activitystarter/fragment/TabbedFragmentActivity.java

```diff
@@ -1,19 +1,20 @@
 package com.example.activitystarter.fragment;
 
 import android.os.Bundle;
-import android.support.v4.app.Fragment;
-import android.support.v4.app.FragmentManager;
-import android.support.v4.app.FragmentPagerAdapter;
-import android.support.v4.view.ViewPager;
-import android.support.v7.app.AppCompatActivity;
 
 import com.example.activitystarter.R;
 
 import activitystarter.ActivityStarter;
 import activitystarter.MakeActivityStarter;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentManager;
+import androidx.fragment.app.FragmentPagerAdapter;
+import androidx.viewpager.widget.ViewPager;
 
 @MakeActivityStarter
-public class TabbedFragmentActivity extends AppCompatActivity {
+public class TabbedFragmentActivity extends AppCompatActivity
+{
```

Comment:
It doesn't look like Kotlin style
https://kotlinlang.org/docs/reference/coding-conventions.html